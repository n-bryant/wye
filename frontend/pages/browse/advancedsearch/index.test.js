import { AdvancedSearchPage } from "./index";

describe("AdvancedSearchPage", () => {
  const props = {
    classes: {
      root: "root"
    }
  };

  it("should render successfully", () => {
    expect(
      toJson(shallow(<AdvancedSearchPage {...props} />))
    ).toMatchSnapshot();
  });
});
