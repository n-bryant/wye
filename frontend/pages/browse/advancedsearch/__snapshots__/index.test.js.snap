// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdvancedSearchPage should render successfully 1`] = `
<div
  className="wye-AdvancedSearchPage root"
>
  <WithWidth(WithStyles(GamesFilterWidget))
    fullSearch={true}
    initialFilters={
      Object {
        "filters": Object {
          "gameFilters": Object {},
        },
        "orderBy": Array [
          "OWNER_COUNT_MAX",
          "USER_RATING",
          "NAME",
        ],
        "sortOrder": "DESC",
      }
    }
    subtitle="Browse all titles"
    title="Advanced Search"
  />
</div>
`;
